lib/client.js

```diff
@@ -58,6 +58,7 @@ bedrock.events.on('bedrock-mongodb.ready', function(callback) {
         var meta = {};
         meta.created = now;
         meta.updated = now;
+        meta.retries = 0;
```

Comment:
`retries` for what? I know, but I have that context ... this should be more specific -- and probably shouldn't be called "retries" unless it's counting down, right? Maybe `messageRetrievalFailures` or something along those lines.